enterprise/app/tap/tap.tsx

```diff
@@ -685,15 +719,13 @@ class InnerTopBar extends React.Component<InnerTopBarProps, InnerTopBarState> {
   private hoveredCommitInfo = React.createRef<HTMLDivElement>();
   private hoveredCommitPointer = React.createRef<HTMLDivElement>();
   // TODO(bduffany): Use a generic tooltip component.
-  private tooltipPortal: HTMLDivElement;
+  private tooltipPortal: HTMLDivElement = document.createElement("div");
```

Comment:
ehhh, rather than doing this outside of the lifecycle, how about just doing ! on the relevant callsites?

Reply:
Done